CMakeLists.txt

```diff
@@ -755,6 +755,10 @@ if (build_tests)
   target_link_libraries(middletest PRIVATE gc)
   add_test(NAME middletest COMMAND middletest)
 
+  add_executable(GC_valloctest tests/GC_valloc.c ${NODIST_SRC})
```

Comment:
Let's drop adding an extra test, GC_valloc() is already tested in gctest. (I know the bdwgc test system is not great, but I resist to redesign it for now.)

Reply:
ok, updated.